docs/start_aml.md

```diff
@@ -0,0 +1,114 @@
+# Use FastAI with Azure Machine Learning!
+This repository has examples showing you how to use FastAI on Azure ML.
+
+# Create an ML workspace
+The [Azure Machine Learning workspace](https://docs.microsoft.com/en-us/azure/machine-learning/overview-what-is-machine-learning-studio) is the top-level resource for the service. It provides you with a centralized place to work with all the artifacts you create. 
+
+You can create an ML workspace by clicking here - **specify a unique resource group name for your Azure subscription!**:
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzureML-fastai%2Fmain%2F.cloud%2Fazuredeploy.json)
+
+More details about deployment in the button used above, Check out [ARM JSON definition](https://github.com/AbeOmor/AzureML-fastai/blob/main/.cloud/azuredeploy.json) and [Azure Machine Learning VM sizes](https://docs.microsoft.com/en-us/rest/api/azureml/workspacesandcomputes/virtualmachinesizes/list)
+
+# Quick FastAI Environment setup
+1) Go to the Azure ML studio (https://ml.azure.com) and find your ML workspace.
+
+2) On the left side, select **Notebooks**. Select any notebook located in the User files section on the left-hand side. If you don't have any notebooks there, first **create a notebook**
+
+![Notebook](images/aml/click_notebook.png)
+
+3) Select the Open terminal icon.
+![Open Terminal](images/aml/open_terminal_full.png)
+
+4) If you don't see the icon, select the ... to the right of the compute target and then select Open terminal .
+![Open Terminal Mini](images/aml/open_terminal.png)
+
+> **PRO-TIP: Copy and Paste in Terminal**
+> * Windows: `Ctrl-c` to copy is supported but use `Shift-insert` to paste.
+> * FireFox/IE may not support clipboard permissions properly.
+> *    Mac OS: `Cmd-c` to copy and `Cmd-v` to paste.
+
+5) Copy the following command to clipboard:
+```shell
+wget https://raw.githubusercontent.com/AbeOmor/AzureML-fastai/main/fastaionAMLCI.sh
```

Comment:
should we host this somewhere more official? maybe fork it to Azure?

Reply:
This doc will appear on course.fast.ai which is the official website for this course. Azure can have a more general document on how to use fast.ai library (beyond the course).

Reply:
I meant the fastaionAMLCI.sh file being downloaded in the shell command

Reply:
"wget https://raw.githubusercontent.com/AbeOmor/AzureML-fastai/main/fastaionAMLCI.sh"

Reply:
Sorry. Didnt realize that comment was for the wget script location. yes. That script should refer to a Azure ML github location rather than individual's github repo.

Reply:
I will add it here: https://github.com/Azure/AzureML-fastai/

Reply:
Just waiting for my PR to be merged there